docs/benchmarking.md

```diff
@@ -0,0 +1,68 @@
+# Benchmarking
+
+## Performance
```

Comment:
Perhaps a redundant header?

Reply:
Probably. But I'm going to have another go at benchmarking now that we have better numbers for small/medium/large feature sets.

Reply:
So, I'll leave it in place for this one.